mithril-aggregator/src/message_adapters/to_certificate_pending_message.rs

```diff
@@ -1,30 +1,24 @@
-use mithril_common::entities::{CardanoDbBeacon, ImmutableFileNumber};
+use mithril_common::entities::CardanoDbBeacon;
 use mithril_common::{
-    entities::{CertificatePending, Signer},
+    entities::{CertificatePending, SignedEntityType, Signer},
     messages::{CertificatePendingMessage, SignerMessagePart},
-    CardanoNetwork,
 };
 
 /// Adapter to turn [CertificatePending] instances into [CertificatePendingMessage].
 pub struct ToCertificatePendingMessageAdapter;
 
 impl ToCertificatePendingMessageAdapter {
     /// Method to trigger the conversion
-    pub fn adapt(
-        certificate_pending: CertificatePending,
-        network: CardanoNetwork,
-        immutable_file_number: ImmutableFileNumber,
-    ) -> CertificatePendingMessage {
-        let beacon = CardanoDbBeacon::new(
-            network.to_string(),
-            *certificate_pending.epoch,
-            immutable_file_number,
-        );
+    pub fn adapt(certificate_pending: CertificatePending) -> CertificatePendingMessage {
+        let beacon = match &certificate_pending.signed_entity_type {
+            SignedEntityType::CardanoImmutableFilesFull(beacon) => beacon.clone(),
+            _ => CardanoDbBeacon::new("", 0, 0),
```

Comment:
We could create a `CardanoDbBeacon::legacy()` instead of `CardanoDbBeacon::new("", 0, 0)` and mark it as deprecated.

Reply:
Good idea but I'm not sure that `legacy`is the right name because it's not a really a legacy value of CardanoDbBeacon. 
I would prefer something like: `no_value` or `empty_value`

Reply:
Yes, `empty` sounds good :+1: